std/haxe/Timer.hx

```diff
@@ -172,8 +172,6 @@ class Timer {
 	public static inline function stamp():Float {
 		#if flash
 		return flash.Lib.getTimer() / 1000;
-		#elseif (neko || php)
```

Comment:
I don't see `Timer.hx` with neko implementation. Did you forget it?
Also, don't remove php conditional, pls )

Reply:
Oh, it's just a duplicated code. Nvm then.

Reply:
Yep, should've called that out.  Both targets are handled by the default case.